Microsoft.Azure.CosmosRepository/src/DefaultRepository.cs

```diff
@@ -182,6 +183,19 @@ await container.UpsertItemAsync<TItem>(value, new PartitionKey(value.PartitionKe
             return updateTasks.Select(x => x.Result);
         }
 
+        public async ValueTask UpdateAsync(string id, string partitionKeyValue,
+            Action<IPatchOperationBuilder<TItem>> builder,
+            CancellationToken cancellationToken = default)

```

Comment:
```suggestion
        public async ValueTask UpdateAsync(
            string id,
            string partitionKeyValue,
            Action<IPatchOperationBuilder<TItem>> builder,
            CancellationToken cancellationToken = default)

```

Reply:
will do this once I have re-jigged the order of the parameters.